src/main/java/org/smartregister/domain/PlanDefinition.java

```diff
@@ -144,7 +147,15 @@ public Long getServerVersion() {
 	public void setServerVersion(Long serverVersion) {
 		this.serverVersion = serverVersion;
 	}
-	
+
+	public Boolean getExperimental() {
```

Comment:
This should be `isExperimental`  since its a boolean

Reply:
also should be primitive boolean

Reply:
Updated method name.